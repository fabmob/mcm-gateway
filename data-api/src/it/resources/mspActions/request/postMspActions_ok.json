{
	"name": "CityScoot",
	"action": "ZONE_OPERATING",
	"isAuthentication": 1,
	"isRefreshAuthentication": 0,
	"isPagination": 0,
	"selector": {
		"key": "test_s",
		"value": "test_s"
	}
}