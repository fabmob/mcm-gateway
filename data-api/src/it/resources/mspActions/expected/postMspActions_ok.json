
{
	"mspActionId": "6dfce23e-5daa-4a40-84cb-a0a9f7feafaf",
	"name": "CityScoot",
	"action": "ZONE_OPERATING",
	"isAuthentication": 1,
	"isRefreshAuthentication": 0,
	"isPagination": 0,
	"selector": {
		"selectorId": "ec0e9a76-f1fe-422c-803b-8332ad957942",
		"key": "test_s",
		"value": "test_s"
	}
}